moabb/utils.py

```diff
@@ -19,3 +20,15 @@ def set_log_level(level="INFO"):
         level=level,
         format="%(asctime)s %(levelname)s %(threadName)s %(name)s %(message)s",
     )
+
+
+def set_download_dir(path):
+    """Set the download directory if required to change from default mne path
+
+    Parameters
+    ----------
+    path : str
+    The new storage location
+
+    """
+    set_config("MNE_DATA", path)
```

Comment:
Maybe you could check before using set_config that:
- the path exist or could be access 
- MNE_DATA is not already configured, if so print a message to say that the user will a to manually move the datasets or that the datasets will be downloading again on the new location.